client/ayon_core/plugins/publish/extract_review.py

```diff
@@ -1228,16 +1228,6 @@ def rescaling_filters(self, temp_data, output_def, new_repre):
         reformat_in_baking = bool("reformated" in new_repre["tags"])
```

Comment:
I was going to push a commit but I don't know what would happen if I changed that line.
```suggestion
        reformat_in_baking = bool("reformatted" in new_repre["tags"])
```

Reply:
Ignoring this in favor of this PR: https://github.com/ynput/ayon-core/pull/285

Reply:
Handled in separate PR https://github.com/ynput/ayon-core/pull/285